server/views/pages/licence/HDC_AP.njk

```diff
@@ -50,11 +50,15 @@
 
    <div id="home-detention-curfew" class="section">
       <p>
-         Your Home Detention Curfew starts on <span class="bold">{{ licence.actualReleaseDate | datetimeToDate or "MISSING - HDC Approved Date/Actual Release Date"}}</span>  and ends on <span class="bold">{{ licence.conditionalReleaseDate | datetimeToDate or "MISSING - Conditional Release Date"}}</span> 
```

Comment:
Removed MISSING text as all values now available - originally used to highlight missing data in the template during development.